release-notes-3.5.1.md

```diff
@@ -0,0 +1,10 @@
+#### Version Number
+${version-number}
+
+#### New Features
+
+#### Patch Fixes Included" and add
```

Comment:
This doesn't look right, check https://github.com/JobService/worker-batch/releases/tag/v3.2.1 and https://github.com/JobService/worker-batch/releases/tag/v3.2.1

Reply:
I followed this Michael's[ comment](https://github.com/Autoscaler/autoscaler/pull/69#discussion_r634195792).
Maybe there are two options to approach a patch release!?